### DISCUSSION TITLE
where to find version 5.3.1.deb for ubuntu 22.04?

### DISCUSSION THREAD

**Discussion by laurent734** [streamlink/streamlink#5264] [Q&A] [answered]
### Description

Good morning ,
I use streamlink on ubuntu server 22.04 with tvheadend but on ubuntu the version is 3.1.1.1 and the plugins are not up to date.
or few have found the 5.3.1.deb file?
THANKS

**Comment by bastimeyer** [comment, marked as answer]
Don't use the Ubuntu packages of Streamlink. They are inherited from Debian and are always outdated. We've explicitly removed Ubuntu from the install docs in #4670 / #4681. Use the AppImages or use python-pip.

- https://streamlink.github.io/install.html#linux-and-bsd
- https://streamlink.github.io/install.html#package-availability
- https://streamlink.github.io/install.html#pypi-package-and-source-code

**Reply by laurent734** [reply]
thank you for your reply.
I managed to make it work with AppImages with tvheadend..
I'm having trouble finding the right command to work with tvheadend but the AppImages remains the simplest...
here is an example for anyone using AppImages in tvheadend...
pipe:///home/hts/streamlink-5.3.1-1-cp311-cp311-manylinux2014_x86_64.AppImage --stdout --hls-live-edge 6 --ringbuffer-size 32M -4 --default-stream 1080p, best --url streamlink https://www.dailymotion.com/video/x5gv5v0
thank you
